src/msg/query_msg.rs

```diff
@@ -48,4 +50,6 @@ pub enum QueryMsg {
     GetCommitments { delegator_addr: String },
     #[returns(QueryEarnPoolResponse)]
     GetLiquidityPools { pool_ids: Option<Vec<u64>>, filter_type: FilterType, pagination: Option<PageRequest> },
+    #[returns(Decimal)]
```

Comment:
it would be better to return a Response msg

Reply:
Got it

Reply:
you forgot to change it here